src/css/style.css

```diff
@@ -372,11 +402,15 @@ td {
     height: 50px;
     width: 100px;
   }
+  .heading {
+    flex-direction: column;
+    justify-content: center;
+  }
```

Comment:
Have you been able to test on a real mobile device? I know flex-direction column has had some weird behavior on actual mobile.

Reply:
I already tested it in my mobile phone, looks fine!
![Screenshot_20201114-002127](https://user-images.githubusercontent.com/31909304/99101687-8739a900-260f-11eb-92d0-a03569f64e33.png)